src/Tasks/Microsoft.NET.Build.Tasks/PublishAssembliesResolver.cs

```diff
@@ -3,14 +3,17 @@
 
 using System;
 using System.Collections.Generic;
+using System.Collections.Concurrent;
 using System.IO;
 using NuGet.Frameworks;
 using NuGet.ProjectModel;
+using NuGet.Packaging.Core;
 
 namespace Microsoft.NET.Build.Tasks
 {
     public class PublishAssembliesResolver
     {
+        private  HashSet<PackageIdentity> _allResolvedPackages = new HashSet<PackageIdentity>();
```

Comment:
Basically, this whole file can be reverted.

---
In reply to: [102985253](https://github.com/dotnet/sdk/pull/890#discussion_r102985253) [](ancestors = 102985253)